src/ui/components/SettingsPage.tsx

```diff
@@ -79,87 +81,110 @@ const SettingsPage: FC = () => {
-          <label className="settings__radio-label">
-            <input
-              className="settings__radio"
+    <Container fluid className="main-content py-4 p-2">
+      <h2 className="light-font fw-bold mt-1 mb-0 ps-3 fs-4 font-monospace">
+        Settings
+      </h2>
+      <hr className="purple-underline ms-3 mb-2 me-2" />
+      <Form>
+        <Form.Group>
+          <Form.Label
+            as="legend"
+            className="light-font fs-5 mt-2 ms-3 mb-3 fw-bold"
+          >
+            Choose experiment collection
+          </Form.Label>
+          <Container className="dark-font mb-3 ms-2 font-monospace">
+            <Form.Check
               type="radio"
+              label={`Live (${LIVE_COLLECTION})`}
               value={LIVE_COLLECTION}
               checked={collectionId === LIVE_COLLECTION}
               onChange={handleCollectionChange}
+              name="collection"
+              id="liveCollection"
+              className="mb-3"
             />
-            Live ({LIVE_COLLECTION})
-          </label>
-          <label className="settings__radio-label">
-            <input
-              className="settings__radio"
+            <Form.Check
               type="radio"
+              label={`Preview (${PREVIEW_COLLECTION})`}
               value={PREVIEW_COLLECTION}
               checked={collectionId === PREVIEW_COLLECTION}
               onChange={handleCollectionChange}
+              name="collection"
+              id="previewCollection"
+              className="mb-2"
             />
-            Preview ({PREVIEW_COLLECTION})
-          </label>
-          <label className="settings__radio-label">
-            <input
-              className="settings__radio"
-              type="radio"
-              value="custom"
-              checked={
-                collectionId !== LIVE_COLLECTION &&
-                collectionId !== PREVIEW_COLLECTION
-              }
-              onChange={handleCollectionChange}
-            />
-            Custom
-            <input
-              type="text"
-              value={customCollection}
-              onChange={handleCustomChange}
-              disabled={
-                collectionId === LIVE_COLLECTION ||
-                collectionId === PREVIEW_COLLECTION
-              }
-            />
-          </label>
-        </div>
-        <hr className="settings__radio-group-line" />
-        <p className="settings__radio-group-title">Re-load experiments</p>
-        <div className="settings__radio-group">
-          <label className="settings__radio-label">
-            <input
-              className="settings__radio"
+            <Container className="d-flex align-items-center ps-0">
+              <Form.Check
+                type="radio"
+                label="Custom"
+                value="custom"
+                checked={
+                  collectionId !== LIVE_COLLECTION &&
+                  collectionId !== PREVIEW_COLLECTION
```

Comment:
Similarly here you could write this as 

```js
![LIVE_COLLECTION, PREVIEW_COLLECTION].includes(collectionId)
```

Makes it easier to change the list of things to check later.